math/dec.go

```diff
@@ -45,13 +45,6 @@ var (
 	smallestDec          = LegacySmallestDec()
 )
 
-// Decimal errors
-var (
-	ErrLegacyEmptyDecimalStr      = errors.New("decimal string cannot be empty")
-	ErrLegacyInvalidDecimalLength = errors.New("invalid decimal length")
-	ErrLegacyInvalidDecimalStr    = errors.New("invalid decimal string")
-)
```

Comment:
Please revert this change, no need to move these errors elsewhere and modify this file.